pkgs/development/embedded/openocd-rp2040/default.nix

```
@@ -0,0 +1,59 @@
+{ lib
+, stdenv
+, fetchgit
+, pkg-config
+, hidapi
+, libftdi1
+, libusb1
+, which
+, libtool
+, autoconf
+, automake
+, texinfo
+, git
+, libgpiod
+}:
+
+stdenv.mkDerivation {
+  pname = "openocd-rp2040";
+  version = "0.11.0";
+  src = fetchgit {
+    url = "https://github.com/raspberrypi/openocd";
+    rev = "8e3c38f78730ce878ff81448bc3f6c1e6bb06e13";
+    sha256 = "sha256-4pY4Kzad53mmnnJDLl2FBDuulUldIo1NcNYzZKSCZts=";
+  };
+
+  nativeBuildInputs = [ pkg-config ];
+
+  buildInputs = [ hidapi libftdi1 libusb1 which libtool autoconf automake texinfo git ]
+    ++ lib.optional stdenv.isLinux libgpiod;
```

```suggestion
  nativeBuildInputs = [
    pkg-config
  ];

  buildInputs = [
    hidapi
    libftdi1
    libusb1
    which
    libtool
    autoconf
    automake
    texinfo
    git
  ] ++ lib.optional stdenv.isLinux libgpiod;
```

just some formatting nitpicks.